crates/rolldown/src/stages/generate_stage/render_chunk_to_assets.rs

```diff
@@ -182,6 +185,18 @@ impl GenerateStage<'_> {
     let mut index_preliminary_assets: IndexInstantiatedChunks =
       IndexVec::with_capacity(chunk_graph.chunk_table.len());
     let chunk_index_to_codegen_rets = self.create_chunk_to_codegen_ret_map(chunk_graph);
+    let render_export_items_index_vec = &chunk_graph
```

Comment:
Pre generate the `export_items_index_vec`, because now render chunk_imports may use other chunk `export_item` info